dScripts/ai/FV/ActNinjaSensei.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+#include "CppScripts.h"
+
+class ActNinjaSensei : public CppScripts::Script {
+	void OnStartup(Entity* self) override;
+	void OnTimerDone(Entity* self, std::string timerName) override;
+private:
+	std::string m_StudentGroup = "Sensei_kids";
+	LOT m_StudentLOT = 2497;
+	std::vector<Entity*> m_Students = {};
```

Comment:
This needs to be a var on the entity - if another of this script is used then things will start to double up again

Reply:
Won't that leak the script in memory though?